config/sidekiq.yml

```diff
@@ -1,5 +1 @@
-:concurrency: 5
-staging:
-  :concurrency: 5
-production:
-  :concurrency: 8
+:concurrency: <%= ENV.fetch('SIDEKIQ_CONCURRENCY', 5).to_i %>
```

Comment:
We need to override the shutdown period to allow sidekiq to gracefully finish jobs before the k8s 30s hard shutdown
```suggestion
:concurrency: <%= ENV.fetch('SIDEKIQ_CONCURRENCY', 5).to_i %>
# Default graceperiod in K8s is 30s
# we can specify longer but will need to increase the terminationGracePeriod
# https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-terminating-with-grace
:timeout: <%= ENV.fetch('SIDEKIQ_TIMEOUT', 25).to_i %>
```

Reply:
committing this with the understanding that it's not being set in the template so it'll default to 25s